.github/workflows/push-bundles.yaml

```diff
@@ -66,3 +69,33 @@ jobs:
         ENSURE_UNIQUE=1 \
         GITHUB_TOKEN=$(curl -s -X POST -H "Authorization: Bearer $(createJWT)" -H "Accept: application/vnd.github+json" "https://api.github.com/app/installations/${APP_INSTALL_ID}/access_tokens" | jq -r .token) \
         hack/update-bundles.sh
+
+    - name: Docker login (quay.io/enterprise-contract)
+      uses: docker/login-action@465a07811f14bebb1938fbed4728c6a1ff8901fc # v2.2.0
+      with:
+        # See also BUNDLE_REPO defined in Makefile
+        registry: quay.io
+        username: ${{ secrets.BUNDLE_PUSH_USER_EC }}
```

Comment:
I guess creating a robot account that could push to both quay.io/hacbs-contract and quay.io/enterprise-contract is not something we can do easily?